app-agent/src/bin/relay_client.rs

```diff
@@ -0,0 +1,108 @@
+use alumet::{agent::AgentBuilder, static_plugins};
+use app_agent::{
+    agent_util, init_logger,
+    options::{
+        cli::{self, ExecArgs},
+        config, AgentModifier,
+    },
+};
+use clap::{Parser, Subcommand};
+
+type AppConfig = config::CommonArgs;
+
+fn main() {
+    let plugins = static_plugins![
+        plugin_rapl::RaplPlugin,
+        plugin_perf::PerfPlugin,
+        plugin_socket_control::SocketControlPlugin,
+        plugin_relay::client::RelayClientPlugin,
```

Comment:
Is it possible to make people using relay_client (or relay_server) unable to desactivate this plugin ?

Reply:
you mean, via the config? why?

Reply:
To prevent user from desactivating the plugin by error